Easydict/App/Localizable.xcstrings

```diff
@@ -4377,6 +4394,202 @@
         }
       }
     },
+    "tips_beep" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Why is there a beep when the selected word is empty (such as dragging in a blank area) in some applications?"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "为什么在某些应用取词为空（空白处拖动等）时会出现提示音？"
+          }
+        }
+      }
+    },
+    "tips_edit_button" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Why does the edit button in the upper right corner flicker when selecting words in some applications?"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "为什么在某些应用取词时，右上角编辑按钮会出现闪烁？"
+          }
+        }
+      }
+    },
+    "tips_might_selecting" : {
+      "extractionState" : "manual",
+      "localizations" : {
+        "en" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Why might selecting an empty word interrupt the music currently playing?"
+          }
+        },
+        "zh-Hans" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "为什么取词为空时可能打断当前播放的音乐？"
```

Comment:
This is the same as before https://github.com/tisfeng/Easydict/pull/499/files#r1601706961